contributors/guide/style-guide.md

```diff
@@ -0,0 +1,653 @@
+  * **Good example:** chair or moderator
+  * **Bad example:** chairman
+* Avoid personal pronouns ("I," "we," "us," "our," and "ours").
+  * In most cases personal pronouns should be avoided as they can lead to
+    confusion regarding who they are referring to. 
+  * **Good example:** The release-team shepherded the successful release of 1.13.
+  * **Bad example:** We shepherded the successful release of 1.13.
+* Address the reader as "you" instead of "we".
+  * Addressing the reader directly using "you" clearly denotes the target.
+    There is no confusion as there would be with "we" or "us".
+  * **Good example:** You will create a new cluster with kubeadm.
+  * **Bad example:** We will create a new cluster with kubeadm.
+* Do not use Latin phrases.
+  * [Latin phrases] can make it difficult for readers not familiar with them to
+    grasp their meaning. 
+  * Some useful alternatives include:
+    | Latin Phrase | Alternative |
+    |:------------:|:-----------:|
+    |     e.g.     | for example |
+    |    et al.    |  and others |
+    |     i.e.     |   that is   |
+    |     via      |    using    |
+  * **Good example:** For example Deployments, ReplicaSets...
+  * **Bad example:** e.g. Deployments, ReplicaSets...
+* Avoid jargons and idioms.
+  * Jargon and idioms tend to rely on regional or tribal knowledge. They make
+    it difficult to understand for both newcomers and those whose native
+    language is something other than English. They should be avoided when
+    possible.
+  * **Good example:** Internally, the kube-apiserver...
+  * **Bad example:** Under the hood the kube-apiserver...
+  * **Good example:** We will start the project in early 2019.
+  * **Bad example:** We will kick off the initative in 2019.
+* If using acronyms, ensure they are clearly defined in the same document.
+  * Acronyms are useful and can make it easier for both the you and the reader. 
+    However, they rely on the assumption that the reader is familiar with the
+    acronym already. This should not be expected of the reader. When
+    introducing a term  that you would like to use an acronym for, use the full
+    form the first time and follow it with the acronym in parentheses.
+  * **Good example:** A _CustomResourceDefinition_ (CRD) extends the Kubernetes
+    API.
+  * **Bad example:** A CRD extends the Kubernetes API.
+
+
+### Moving a Document
+
+* When moving a document, leave a tombstone file with a removal date in it's
+  place.
+  * Tombstones function as a pointer and give users a time to update their own
+    documentation and bookmarks. Their usefulness is time-bounded and should be
+    removed when they would logically no longer serve their purpose.
+    ```markdown
+    This file has moved to https://git.k8s.io/community/contributors/guide/README.md.
+
+    This file is a placeholder to preserve links.  Please remove after 3 months or the release of kubernetes 1.10, whichever comes first.
```

Comment:
Could we use an exact date in the format suggested at Dates and Time (e.g. December 13, 2018) instead of 3 months?

Reply:
updated with explicit date in the example :)